chef/cookbooks/keystone/libraries/helpers.rb

```diff
@@ -3,8 +3,11 @@ def self.service_URL(node, host, port)
     "#{node[:keystone][:api][:protocol]}://#{host}:#{port}"
   end
 
-  def self.versioned_service_URL(node, host, port)
-    service_URL(node, host, port) + '/' + node[:keystone][:api][:version] + '/'
+  def self.versioned_service_URL(node, host, port, version=nil)
```

Comment:
Probably. But I found this a bad idiom. AFAIU the default parameter is only evaluated once, we the module is first loaded. So this means that if someone change the global value of `node`, this change will be no reflected in the parameter.  This can be very confusing.

I know that no one will change the value, but at least I try to express that the value is always readed from the global `node` every time this is called.